src/dev_server/index.tsx

```diff
@@ -8,10 +8,13 @@ import { fetchMainVersionMap, getMainUrl } from './url_resolver';
 
 let config: Config;
 const RETRY_TIMER = 5000;
+const LOGS_ENABLED = false;
```

Comment:
It might be worth it to move this into `config` as an optional value. As it is, the only way to turn this on is to directly modify this file in your local node_modules. (I know I'm the one who added it though haha 😅 )

Reply:
noted.